src/config.js

```diff
@@ -171,6 +171,7 @@ let config = {
   MFE_CONFIG_API_URL: process.env.MFE_CONFIG_API_URL,
   APP_ID: process.env.APP_ID,
   SUPPORT_URL: process.env.SUPPORT_URL,
+  PARAGON_THEME_URLS: process.env.PARAGON_THEME_URLS ? JSON.parse(process.env.PARAGON_THEME_URLS) : {},
```

Comment:
If the environment variable `PARAGON_THEME_URLS` has invalid JSON, is there a chance this `JSON.parse(...)` would throw an error? Might it be worth implementing a helper function to wrap the `JSON.parse(...)` with a `try / catch` for improved error handling?

Reply:
That sounds like a great plan. then this line could just be
```js
PARAGON_THEME_URLS: parseParagonThemeUrls(process.env.PARAGON_THEME_URLS),
```

and the function could handle all the logic (both `process.env.PARAGON_THEME_URLS` being falsy and invalid json)